pythonosc/parsing/osc_types.py

```diff
@@ -199,6 +200,42 @@ def get_float(dgram, start_index):
     raise ParseError('Could not parse datagram %s' % e)
 
 
+def write_double(val):
+  """Returns the datagram for the given double parameter value
+
+  Raises:
+    - BuildError if the double could not be converted.
+  """
+  try:
+    return struct.pack('>d', val)
+  except struct.error as e:
+    raise BuildError('Wrong argument value passed: {}'.format(e))
+
+
+def get_double(dgram, start_index):
+  """Get a 64-bit big-endian IEEE 754 floating point number from the datagram.
+
+  Args:
+    dgram: A datagram packet.
+    start_index: An index where the double starts in the datagram.
+
+  Returns:
+    A tuple containing the double and the new end index.
+
+  Raises:
+    ParseError if the datagram could not be parsed.
+  """
+  try:
+    if len(dgram[start_index:]) < _DOUBLE_DGRAM_LEN:
+      raise ParseError('Datagram is too short')
+    return (
+        struct.unpack('>d',
+                      dgram[start_index:start_index + _DOUBLE_DGRAM_LEN])[0],
+        start_index + _DOUBLE_DGRAM_LEN)
+  except (struct.error, TypeError) as e:
+    raise ParseError('Could not parse datagram %s' % e)
```

Comment:
Please use raise ParseError('Could not parse datagram {}'.format(e))
for consistency